### PR TITLE
Implement in "progress table"

### LINKED ISSUE #6: Implement 'In progress' table
Implement `In progress` table on `/main/queue` page:

Image

[Figma](https://www.figma.com/file/cRvtq7N0IQ2aRJiwD7rMeu/Alchemestry?node-id=249-4272&t=FzBSniZ7JwQR60as-0)

### PR DESCRIPTION
**Pull request by Romazes**
# Description
Implement part of progress table in app. I tried to keep all styling code convention.
Some codes' part have carried out in independent components.
# Related Issue
close #6

# Result
image

# Suggestion
Please review my code and give me feedback.

# Questions
1. Why doesn't it work the deployment in **Vercel** ?

### CODE REVIEW

**Comment by gymnasy55** [comment]
try run `yarn lint --fix`

**Comment by Romazes** [comment]
> try run `yarn lint --fix`

It helped.

File: src/components/ProgressTable/index.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+import { ProgressbarCard } from './ProgressbarCard';
+
+import { Title } from '../Title';
+
+import ProgressBarLevel1 from '@/../public/images/progressbar-bg-level-1.png';
+import ProgressBarLevel2 from '@/../public/images/progressbar-bg-level-2.png';
+import ProgressBarLevel3 from '@/../public/images/progressbar-bg-level-3.png';
+
+export const Progressbar = () => {
```
Comment (gymnasy55) [inline src/components/ProgressTable/index.tsx]: Rename to `ProgressBar`

File: src/components/ProgressTable/index.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  return (
+    <>
+      <Title>Progress</Title>
+      <div className="ml-9 mt-9 flex gap-x-9">
```
Comment (gymnasy55) [inline src/components/ProgressTable/index.tsx]: add `flex-wrap` style so flex will be responsive

File: src/components/ProgressTable/index.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        <ProgressbarCard
```
Comment (gymnasy55) [inline src/components/ProgressTable/index.tsx]: Create array with such objects and then just map over them

Example:
```js
const array = [{ title: "A", image: Img, amount: 228, maxAmount: 1337 }];

return array.map(({ title, image, amount, maxAmount }) => <ProgressbarCard key={title} title={title} .... /> )

File: src/components/ProgressTable/ProgressbarCard.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import type { StaticImageData } from 'next/image';
+import Image from 'next/image';
+import type { FC } from 'react';
+import React from 'react';
+
+export const ProgressbarCard: FC<{
```
Comment (gymnasy55) [inline src/components/ProgressTable/ProgressbarCard.tsx]: Rename to `ProgressBarCard`

Comment (gymnasy55) [inline src/components/ProgressTable/ProgressbarCard.tsx]: create interface for props

Example:
```ts
interface ProgressBarCardProps {
  title: string;
  image: StaticImageData;
  currentAmount: number;
  maxAmount: number;
}

const ProgressBarCard: FC<ProgressBarCardProps> = () => { ... }
```

File: src/components/Title/index.tsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+import { StarIcon } from '@/components/icons/StarIcon';
+
+export const Title: FC<{ children: string }> = ({ children }) => {
+  return (
+    <div className="flex h-[48px]">
+      <StarIcon size={35} />
+      <div className="relative text-3xl font-bold capitalize">
+        {children}
+        <span className="absolute bottom-0 left-0 w-1/2 bg-gradient-to-r from-primary to-primary-dark p-[2px]"></span>
```
Comment (gymnasy55) [inline src/components/Title/index.tsx]: Do not create dom element for such spans, just use `before` or `after` css pseudo-classes. 

Example from FAQ page (staging branch):
```js
<div className="relative text-[32px] font-bold before:absolute before:-bottom-[4px] before:h-[3px] before:w-[180px] before:bg-ternary">
  Questions & Answers
</div>
```

**Review by gymnasy55 — CHANGES_REQUESTED** [review]

File: src/components/ProgressTable/index.tsx (hunk 2 above)
Comment (Romazes) [reply src/components/ProgressTable/index.tsx]: It looks ugly :( 

image

File: src/components/ProgressTable/index.tsx (hunk 3 above)
Comment (Romazes) [reply src/components/ProgressTable/index.tsx]: Does the best practice do mapping in jsx markup ?

**Review by gymnasy55 — APPROVED** [review]

**Pull request merged**